src/server/src/routes/users.js

```diff
@@ -10,58 +10,63 @@ dotenv.config();
 
 const router = express.Router();
 
+const mg = mailgun({
+  apiKey: process.env.MAILGUN_APIKEY,
+  domain: process.env.MAILGUN_DOMAIN,
+});
+
+// Hash passwords
+const hashPassword = async (password) => {
+  const salt = await bcrypt.genSalt(10);
```

Comment:
magic number...

Reply:
You can leave it and write a comment why "10" is a good choice

Reply:
fixed